conf/test_busco_auto.config

```diff
@@ -16,8 +16,8 @@ params {
 
     // Limit resources so that this can run on GitHub Actions
     max_cpus   = 2
-    max_memory = 6.GB
-    max_time   = 48.h
+    max_memory = '6.GB'
+    max_time   = '6.h'
```

Comment:
Yes, the template specifies `'6.h' ` since 2.0 already... see https://github.com/nf-core/tools/blob/e563811adb292b07ec933a52611e937ce9ef5481/nf_core/pipeline-template/conf/test.config#L18-L20